.travis.yml

```diff
@@ -15,6 +15,7 @@ env:
   - DOCKERFILE_DIR=nginx
   - DOCKERFILE_DIR=redis
   - DOCKERFILE_DIR=httpd
+  - DOCKERFILE_DIR=php
```

Comment:
Please rebase to get the current sorted list of images for the Travis CI config.